picasso/src/test/java/com/squareup/picasso3/NetworkRequestHandlerTest.java

```diff
@@ -203,6 +204,26 @@
     assertThat(latch.await(10, SECONDS)).isTrue();
   }
 
+  @Test public void shouldHandleSchemeInsensitiveCase() {
+    String[] schemes = {
+            "http",
+            "https",
+            "HTTP",
+            "HTTPS",
+            "HTtP",
+    };
+
+    for (String scheme : schemes) {
+      final Uri uri = URI_1.buildUpon().scheme(scheme).build();
+      final Boolean shouldHandle = networkHandler.canHandleRequest(
+              TestUtils.mockRequest(uri)
+      );
+
+      assertThat(shouldHandle).isTrue();
```

Comment:
can probably inline here.

Reply:
i didn't inline here for legibility and column limit (that's why i used final btw, i always think that the compiler will inline "one reference/usage final" variables - but i know that even without `final` the compiler can do that), anyway, i will inline it